### PR TITLE
Release notes for Desktop Edge 2.3.0.1

### PR DESCRIPTION
**Pull request by usha-mandya**
### Proposed changes

Release notes for Docker Desktop Edge 2.3.0.1

### Unreleased project version (optional)

Docker Desktop Edge 2.3.0.1

### CODE REVIEW

File: docker-for-mac/edge-release-notes.md [hunk 1]
```diff
@@ -12,6 +12,14 @@ For information about Stable releases, see the [Stable release
 notes](release-notes). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.3.0.1
+2020-04-DATE
+
+> Download - Add URL
+
+### Bug fixes and minor changes
+Contains minor improvements.
```
Comment (stephen-turner) [inline docker-for-mac/edge-release-notes.md:21]: I know this history of this, but to be honest it sounds weird, and raises more questions than it answers. Let's brainstorm it in today's standup.

Comment (stephen-turner) [reply docker-for-mac/edge-release-notes.md:21]: If we decide to take https://github.com/docker/pinata/pull/13804, we could put that in this section.

Comment (djs55) [reply docker-for-mac/edge-release-notes.md:21]: What do you think of the wording here: https://github.com/docker/pinata/pull/13809
```
- Fixed a bug starting and stopping a compose application from the UI when the path contains whitespace.
```

Comment (djs55) [reply docker-for-mac/edge-release-notes.md:21]: I've merged your suggestion into the changelog in docker/pinata#13809:
```
Fixed a bug that caused starting and stopping of a Compose application from the UI to fail when the path contains whitespace.
```

**Pull request merged**